src/AssociationRegistry/Acties/WijzigBasisgegevens/WijzigBasisgegevensCommandHandler.cs

```diff
@@ -16,11 +16,20 @@ public class WijzigBasisgegevensCommandHandler
         HandleKorteNaam(vereniging, message.Command.KorteNaam);
         HandleKorteBeschrijving(vereniging, message.Command.KorteBeschrijving);
         HandleStartdatum(vereniging, message.Command.Startdatum, clock);
+        HandleHoofdactiviteitenVerenigigsloket(vereniging, message.Command.HoofdactiviteitenVerenigingsloket);
 
         var result = await repository.Save(vereniging, message.Metadata);
         return CommandResult.Create(VCode.Create(message.Command.VCode), result);
     }
 
+    private static void HandleHoofdactiviteitenVerenigigsloket(Vereniging vereniging, HoofdactiviteitVerenigingsloket[]? hoofdactiviteitenVerenigingsloket)
+    {
+        if (hoofdactiviteitenVerenigingsloket is null)
+            return;
+
+        vereniging.WijzigHoofdactiviteitenVerenigigsloket(hoofdactiviteitenVerenigingsloket);
+    }
+
     private static void HandleStartdatum(Vereniging vereniging, Startdatum? startdatum, IClock clock)
```

Comment:
Do we want to move null checks to the domain? This seems unnecessary at this point, and something the aggregate could handle.